src/role_scout.js

```diff
@@ -57,6 +80,7 @@ roles.scout.execute = function(creep) {
         creep.memory.skip.push(creep.memory.search.target);
         delete creep.memory.scoutSkip;
       } else {
+        checkForDefender(creep);
```

Comment:
May be better drop a defender on all rooms linked to reserved one no? the issue i supose with your logic is that get linear distance may focus rooms which are further than expected if some exits are closed. could use search.level I supose with level <= 2 but it will defend all roms around level 1.

Reply:
Defenders are called for rooms with linear distance 1 (https://github.com/TooAngel/screeps/pull/402/files#diff-362993a173ef9b2bb120bc815a19a4cfR41). Yes in some cases this can be a couple of rooms away, but for the start I think it is fine before getting more sophisticated.